guides/instrumenting-a-module.md

```diff
@@ -16,7 +16,7 @@ function takes four arguments:
 The signature is `ao.instrument (span, fn, config, callback)`.
 
 - `span` is either the string name of the span to be created or a function that returns the span to use.
-- `fn` is a function that will run the function to be instrumented.
+- `runner` is a function that will run the function to be instrumented.
```

Comment:
The function signature still has `fn` so now this `runner` does not refer to anything.

Reply:
thanks - done.